docs/src/pages/docs/monoids/First.md

```diff
@@ -138,6 +138,40 @@ concat(concat(a, b), c)
 //=> First( Just "c" )
 ```
 
+#### equals
+
+```haskell
+First a ~> b -> Boolean
+```
+
+Used to compare the underlying values of (2) `First` instances for equality by value, equals takes any given argument and returns `true` if the passed argument is a `First` with an underlying value equal to the underlying value of the `First` the method is being called on. If the passed argument is not a `First` or the underlying values are not equal, equals will return `false`.
```

Comment:
There's a convention in place elsewhere in the docs to start a document with a description. See for example https://github.com/evilsoft/crocks/blob/master/docs/src/pages/docs/monoids/Any.md which begins "`Any` is a `monoid` which..."

I think it would be good to stick to that convention here, too.

Reply:
@bennypowers 
I am confused. This is just an addition to an existing document.
It looks like this is in the proper format starting on line 1.

Reply:
You're right good catch!